adapters/Vungle/VungleAdapter/VungleRouterConsent.m

```diff
@@ -13,23 +13,24 @@
 // limitations under the License.
 
 #import "VungleRouterConsent.h"
-#import <VungleSDK/VungleSDK.h>
-
-static VungleConsentStatus currentConsentStatus = 0;
+#import <VungleAdsSDK/VungleAdsSDK.h>
 
 @implementation VungleRouterConsent
-+ (BOOL)updateConsentStatus:(VungleConsentStatus)consentStatus {
-  VungleSDK *sdk = VungleSDK.sharedSDK;
-  if (consentStatus > 0) {
-    currentConsentStatus = consentStatus;
-    [sdk updateConsentStatus:consentStatus consentMessageVersion:@""];
-    return YES;
-  }
-  return NO;
+
++ (void)updateGDPRStatus:(BOOL)optIn {
+  [VunglePrivacySettings setGDPRStatus:optIn];
+}
+
++ (void)updateCCPAStatus:(BOOL)optIn {
+  [VunglePrivacySettings setCCPAStatus:optIn];
+}
+
++ (void)updateCOPPAStatus:(BOOL)optIn {
+  [VunglePrivacySettings setCOPPAStatus:optIn];
```

Comment:
What if the publisher hasn't specified COPPA value?
https://developers.google.com/admob/ios/targeting#child-directed_setting

Should we just not set the COPPA value in the Vungle SDK?

Reply:
I will remove this wrapper around the Vungle COPPA. There was a previous comment somewhere that the publisher should use the Google SDK API to set the COPPA flag and the adapter would attempt to pull the childDirectedTreatment flag from the Google SDK on every load request.
https://github.com/Vungle/googleads-mobile-ios-mediation/blob/7.0.x_pre_release/adapters/Vungle/VungleAdapter/GADMediationAdapterVungle.m#L111

But I just moved the method away from the public header at the time.